### PR TITLE
Move to GH actions for testing

### LINKED ISSUE #8: move CI off Travis to Github Actions
e.g. https://github.com/fonttools/skia-pathops/blob/master/.github/workflows/ci.yml

### PR DESCRIPTION
**Pull request by simoncozens**
And build wheels too, bonus! Fixes #8

### CODE REVIEW

File: .github/workflows/wheels.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    name: Build wheel ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu-20.04", "windows-2019", "macos-11" ]
+    env:
+      CIBW_ARCHS_MACOS: x86_64 arm64 universal2
+      CIBW_TEST_COMMAND: pytest {project}/tests
+      CIBW_TEST_REQUIRES: pytest
```
Comment (anthrotype) [inline .github/workflows/wheels.yml:16]: have you not specified the test-command and test-requires in the pyproject.toml file under `[tool.cibuildwheel]`?

File: .github/workflows/wheels.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+jobs:
+  build:
+    name: Build wheel ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu-20.04", "windows-2019", "macos-11" ]
+    env:
+      CIBW_ARCHS_MACOS: x86_64 arm64 universal2
```
Comment (anthrotype) [inline .github/workflows/wheels.yml:14]: why do we need all three of them? If we do universal2, we should not need the other specific ones, no?

Comment (anthrotype) [reply .github/workflows/wheels.yml:14]: I guess the univerasal2 version of Python (from python.org) can be run in native arm64 mode or in x86_64 compatibility mode, and depending on that pip should select the appropriate wheel.. But I'm actually not sure what is the priority among those three. If it's "prefer the more specific one", then the universal2 will never get used? I can't know for sure as I don't have an M1 machine to test with

Comment (anthrotype) [reply .github/workflows/wheels.yml:14]: it looks like projects such as numpy build and publish all three of them to PyPI, we may as well do the same

**Review by anthrotype — APPROVED** [review]

**Comment by anthrotype** [comment]
FYI, I deleted the travis and appveyor webhooks in the repo settings. Next PR we should no longer see the "Appveyor build failed"

File: .github/workflows/wheels.yml (hunk 1 above)
Comment (simoncozens) [reply .github/workflows/wheels.yml:16]: Yes, but for some reason it didn't cause cibuildwheel to run the tests.

File: .github/workflows/wheels.yml (hunk 2 above)
Comment (simoncozens) [reply .github/workflows/wheels.yml:14]: I might be wrong, but I think that universal wheels are only supported on macOS 11.

**Pull request merged**